contributing_to_docs/tools_and_setup.adoc

```diff
@@ -91,104 +80,13 @@ $ docker run --rm -it -v `pwd`:/docs:Z quay.io/openshift-cs/asciibinder asciibin
 filename as the original `.adoc` file you edited, but with an
 `.html` extension.
 
-=== Install the required software dependencies on a Linux system
-The following instructions describe how to install all the required tools to do
-live content editing on a Fedora Linux system.
-
-1. Install the _RubyGems_ package with `yum install rubygems`
-+
-[NOTE]
+[IMPORTANT]
 ====
-On certain systems, `yum` installs an older version of RubyGems that can cause issues. As an alternative, you can install RubyGems by using RVM. The following example is referenced from the link:https://rvm.io/rvm/install[RVM site]:
+Previously, installing AsciiBinder directly on your system was recommended. However, on recent Linux distribution releases, AsciiBinder does not work because of wrong Ruby infrastructure versions.
```

Comment:
```suggestion
Previously, installing AsciiBinder directly on your system was recommended. However, on recent Linux distribution releases, AsciiBinder does not work because of Ruby version incompatibility.
```